playbooks/install_os.yml

```diff
@@ -4,6 +4,21 @@
   hosts: all
   strategy: linear
   become: true
+  vars:
```

Comment:
Is the plan to document that users of the playbook should override `timesync_ntp_servers` as well as have install_yamls generate EDPM ansible vars which can override these values?

Reply:
Overrides in install_yamls only. These are reasonable defaults and I don't think we should have users overriding them unless absolutely necessary.

Reply:
I agree these are reasonable defaults. I know for sure though there will be Red Hatters and customers not using install_yamls who want to override them.

Reply:
Nope, I disagree. I'm putting my sysadmin hat here. :D

The vast majority of our customers environments have internal NTP servers (and most of the time servers can't even connect to internet) so I'm 100% sure they need to customize them.

I'd leave the `timesync_ntp_servers` var here as "sane default" (and self-explaining example) but we need to document how to override it, even linking the role documentation.

Reply:
Well, we don't really have documentation for individual playbooks yet. I suppose it's time to start one. In the transitionary period, before install_yamls is patched, we'll need to support three way resolution for defaults, at least for the ntp servers list.